test/controllers/products_controller_test.rb

```diff
@@ -238,21 +241,27 @@
         end
       end
 
-      describe "update as Guest" do
-        it "should respond with found and error message to prompt user to login" do
+      describe 'update as Guest' do
+        it 'should respond with found and error message to prompt user to login' do
           patch product_path(Product.first.id)
 
           must_respond_with :found
-          expect(flash[:error]).must_equal "You must be logged in to do this action"
+          expect(flash[:error]).must_equal 'You must be logged in to do this action'
```

Comment:
Random: was changing all the double quotes to single quotes and {}s to do ends intentional? Both valid; I'm just curious 🤷🏼‍♀️

Reply:
I have some extension in VS code that does that, and I can't figure out which one it is/how to turn it off haha